### ISSUE TITLE
Packages being reported as out of date when they're in fact the latest

### ISSUE THREAD

**Issue by ffxsam** [meteor/meteor#6701]
Meteor 1.3. My packages are all completely current, yet the list shows several are erroneously out of date:

```
ecmascript             0.4.1* Compiler plugin that supports ES2015+ in all .j...
es5-shim               4.5.8* Shims and polyfills to improve ECMAScript 5 sup...
fourseven:scss         3.4.1  Style with attitude. Sass and SCSS support for ...
jquery                 1.11.6* Manipulate the DOM using CSS selectors
kadira:flow-router     2.11.0  Carefully Designed Client Side Router for Meteor
meteor-base            1.0.2* Packages that every Meteor app needs
mobile-experience      1.0.2* Packages for a great mobile user experience
mongo                  1.1.5* Adaptor for using MongoDB and Minimongo over DDP
random                 1.0.7* Random number generator and utilities
react-meteor-data      0.2.7  React mixin for reactively tracking Meteor data
reactive-dict          1.1.5* Reactive dictionary
reactive-var           1.0.7* Reactive variable
standard-minifier-css  1.0.4* Standard css minifier used with Meteor apps by ...
standard-minifier-js   1.0.4* Standard javascript minifiers used with Meteor ...
static-html            1.0.5* Define static page content in .html files
tracker                1.0.11* Dependency tracker to allow reactive callbacks
```

**Comment by ramirez530** [comment]
Ramirez044@Yahoo.com

**Comment by abernix** [comment]
It's a bit confusing, but this is because technically Meteor 1.3.1 is out as of yesterday, but it is not currently the `recommended` version so automatic upgrading to all of those (core) packages, will not happen automatically (note the additional message above that didn't output).  It *is* considered stable though, so if you want to try it you can do `meteor update --release 1.3.1` and this will be resolved for you – otherwise this will happen until next week (or whenever they switch it to "recommended").

Full announcement [here in the forums](https://forums.meteor.com/t/announcing-meteor-1-3-1/20716)

**Comment by stubailo** [comment]
This will be fixed when 1.3.1 is recommended. We might not do this style of release where we wait to recommend a release again in the future, so this hopefully won't come up again.
